modules/EmbedPlayer/resources/mw.EmbedPlayerNative.js

```diff
@@ -789,7 +789,9 @@
 							vid.play();
 						} else {
 							_this.removeBlackScreen();
-							vid.play();
+							if (!mw.isIOS()) {
+								vid.play();
```

Comment:
From the comment above it actually sounds like we need this play event and something else should pause the player.

Reply:
I actually have added this play event in here:
https://github.com/kaltura/mwEmbed/pull/4001/files

Reply:
So you should have probably added this original play call with a condition or in another place